src/citrine/resources/data_concepts.py

```diff
@@ -632,10 +637,81 @@ def filter_by_name(self, name: str, exact: bool = False,
         response = self.session.get_resource(
             # "Ignoring" dataset because it is in the query params (and required)
             self._get_path(ignore_dataset=True) + "/filter-by-name",
-            params=params,
-        )
+            params=params)
         return [self.build(content) for content in response["contents"]]
 
+    def list_by_name(self, name: str, exact: bool = False,
+                     forward: bool = True, per_page: int = 100) -> Iterator[DataConcepts]:
+        """
+        Get all objects with specified name in this dataset.
+
+        Parameters
+        ----------
+        name: str
+            case-insensitive object name prefix to search.
+        exact: bool
+            Set to True to change prefix search to exact search (but still case-insensitive).
+            Default is False.
+        forward: bool
+            Set to False to reverse the order of results (i.e. return in descending order).
+        per_page: int
+            Controls how many results are fetched per request. Has no impact on the size of
+            the result set but will have some effect on performance. Lower gets results
+            faster but increases the total number of requests needed to fetch the entire
+            result set, which increases total latency across fetching the entire result set.
```

Comment:
Since this is a public docstring, less definitive language about performance, and more helpful language may be motivated, here. 

"Controls the number of results fetched with each http request to the backend. Typically, this is set to a sensible default and should not be modified. Consider modifying this value only if you find this method is unacceptably latent."